test/unit/devices_repository_test.dart

```diff
@@ -178,7 +178,7 @@ void main() {
         thermostatDevice!.device!.traits.asList());
     final thermostatTrait = convertedValue.whereType<ThermostatTrait>().first;
 
-    expect(thermostatTrait.name, 'thermostat_setting');
+    expect(thermostatTrait.name, 'THERMOSTAT_SETTING');
```

Comment:
would it make sense here to compare it against `GTraitName.THERMOSTAT_SETTING.name` instead of `THERMOSTAT_SETTING`?

Reply:
Yes it sure would